src/Symfony/Component/Form/AbstractTypeExtension.php

```diff
@@ -45,4 +46,18 @@ public function finishView(FormView $view, FormInterface $form, array $options)
     public function configureOptions(OptionsResolver $resolver)
     {
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getExtendedType()
+    {
+        if (!method_exists($this, 'getExtendedTypes')) {
+            throw new LogicException(sprintf('You need to implement the static getExtendedTypes() method when implementing the %s in %s.', FormTypeExtensionInterface::class, static::class));
```

Comment:
shouldn't it rather use `static::getExtendedType()` and trigger a deprecation notice for BC? I mean, this breaks for anyone not having upgrading and not yet implementing the new method, isn't it?

Reply:
Nope, before PHP would have prevented that case because you had to implement the `getExtendedType()` method yourself.

Reply:
of course, thanks